go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/openshift/cluster-monitoring-operator
 
-go 1.22
+go 1.22.0
+
+toolchain go1.22.4
```

Comment:
Will we need to adjust any of our PR checks to guard against accidentally bumping the toolchain directive?
Or is there anything to consider how ART deals with this?